conversion_manager.py

```diff
@@ -0,0 +1,25 @@
+import data_sources
+import exceptions
+import factory
+from xsdata.formats.dataclass.serializers import XmlSerializer
+
+
+class ConversionManager:
+    def __init__(self, serializer: XmlSerializer, dispatcher: data_sources.dispatcher.DataSourceDispatcher) -> None:
+        self.serializer = serializer
+        self.dispatcher = dispatcher
+
+    def convert(self, input_filename: str, output_filename: str, data_source: str) -> None:
+        verification_dto = self.dispatcher.get_data_generator_by_source_name(input_filename, data_source)
```

Comment:
It's not a `verification_dto`. It's a generator which will iterate over multiple DTO's.

Reply:
I'd also define `data_source` as a first argument.